api/health.go

```diff
@@ -285,8 +288,54 @@ func normalizeProperty(unitName string, p map[string]interface{}, d DCOSHelper)
 	}
 }
 
+func updateSystemMetrics() (sysMetrics sysMetrics, err error) {
```

Comment:
You could possibly simplify/clarify this block a bit by adding the types from psutil lib directly to your parent struct like:

```
type sysMetrics struct {
	Memory *mem.VirtualMemoryStat  `json:"memory"`
	CPU    []cpu.CPUInfoStat       `json:"cpu"`
	Host   *host.HostInfoStat      `json:"host"`
	Load   *load.LoadAvgStat       `json:"load"`
	Disk   *disk.DiskUsageStat     `json:"disk"`
```

And then declaring getter funcs on it:

```
func HostInfo() *host.HostInfoStat {
	h, _ := host.HostInfo()
	return h
}

func HostMemory() *mem.VirtualMemoryStat {
	memory, _ := mem.VirtualMemory()
	return memory
}

func HostCpu() []cpu.CPUInfoStat {
	c, _ := cpu.CPUInfo()
	return c
}

func HostLoad() *load.LoadAvgStat {
	l, _ := load.LoadAvg()
	return l
}
```